sys-utils/rfkill.c

```diff
@@ -232,28 +232,32 @@ static int rfkill_event(void)
 	struct rfkill_event event;
 	struct timeval tv;
 	char date_buf[ISO_BUFSIZ];
-	struct pollfd p;
+	struct pollfd p[2];
 	int fd, n;
 
 	fd = rfkill_open(1, 0);
 	if (fd < 0)
 		return fd;
 
 	memset(&p, 0, sizeof(p));
-	p.fd = fd;
-	p.events = POLLIN | POLLHUP;
+	p[0].fd = fd;
+	p[0].events = POLLIN | POLLHUP;
+	p[1].fd = 1; /* stdout */
```

Comment:
Pls, use STDOUT_FILENO macro rather than 1.

Reply:
In other places in util-linux we use macros rather magic numbers:
```
       enum {   
                                POLLFD_RFKILL = 0,
                                POLLFD_STDOUT
        };       

        p[POLLFD_RFKILL].fd = fd;
        ... etc.
````

Reply:
Thank you, wasn't aware of those macros.

Would it be okay to also add `POLLFD_COUNT` to the enum, and use that for the array definition instead of `p[2]`?

Reply:
One more question: isn't the `= 0` in the enum definition redundant?

Reply:
Yes, POLLFD_COUNT is fine.

Yes =0 is redundant. (I sometimes use it to make it obvious for readers and to make it visible that zero is important here -- but you can avoid it ;-))